src/test/java/com/wikia/webdriver/common/core/configuration/EnvType.java

```diff
@@ -25,4 +25,8 @@
   public String getDomain() {
     return Configuration.getForceFandomDomain() ? fandomDomain : wikiaDomain;
   }
+
+  public String getDomain(String currentURL) {
```

Comment:
IMO this is not a good place for this method, since it's not part of Configuration. It'd be better to move it to e.g. BasePageObject

Reply:
@nikodamn it's an overload of a method above

Reply:
Yeah, but it's not using Configuration anymore...